cmd/netgoal/generate.go

```diff
@@ -199,7 +199,10 @@ func generateNetworkTemplate(templateFilename string, wallets, relays, nodeHosts
 		newNode.NodeNameMatchRegex = ""
 		newNode.FractionApply = 0.0
 		newNode.Name = name
-		newNode.IsRelay = true
+		if newNode.NetAddress == "" {
```

Comment:
I think that you don't want to have these workarounds; you want to make sure that the relayTemplates is configured correctly. If not, we need to fix it there.

Reply:
yeah, the other option would to be to fail with error message if a "relay" template failed to set NetAddress. I'll work that up.

Reply:
Yeah - fail if netaddress not specified for relays - don't default.

Reply:
returns an error now